amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresher.java

```diff
@@ -162,6 +161,16 @@ public DynamoDBLeaseRefresher(final String table, final DynamoDbAsyncClient dyna
     @Override
     public boolean createLeaseTableIfNotExists(@NonNull final Long readCapacity, @NonNull final Long writeCapacity)
             throws ProvisionedThroughputException, DependencyException {
+        // DynamoDB is now created in PayPerRequest billing mode by default. Keeping this for backward compatibility.
+        return createLeaseTableIfNotExists();
```

Comment:
Let's not change the old code path - this will change the behavior for any customer classes that use this.

Reply:
addressed this.